etna/transforms/feature_selection/filter.py

```diff
@@ -3,12 +3,15 @@
 
 import pandas as pd
 
+from etna.transforms.base import DymmyInColumnMixin
 from etna.transforms.base import Transform
 
 
-class FilterFeaturesTransform(Transform):
+class FilterFeaturesTransform(Transform, DymmyInColumnMixin):
```

Comment:
Do we really need DymmyInColumnMixin?

Reply:
This mixin indicates the classes with fake or constant in_column, I guess we can leave it now and remove in the future if we do't need it

Reply:
I though there would be some additional logic in in_column generation for feature_selection, however there isn't